app/models/bgs_dependents/relationship.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class Relationship < Base
+    def initialize(proc_id)
+      @proc_id = proc_id
+    end
+
+    def params_for_686c(participant_a_id, dependent)
+      {
+        vnp_proc_id: @proc_id,
+        vnp_ptcpnt_id_a: participant_a_id,
+        vnp_ptcpnt_id_b: dependent[:vnp_participant_id],
+        ptcpnt_rlnshp_type_nm: dependent[:participant_relationship_type_name],
+        family_rlnshp_type_nm: dependent[:family_relationship_type_name],
+        event_dt: format_date(dependent[:event_date]),
+        begin_dt: format_date(dependent[:begin_date]),
+        end_dt: format_date(dependent[:end_date]),
+        marage_state_cd: dependent[:marriage_state],
+        marage_city_nm: dependent[:marriage_city],
+        marage_trmntn_state_cd: dependent[:divorce_state],
+        marage_trmntn_city_nm: dependent[:divorce_city],
+        marage_trmntn_type_cd: dependent[:marriage_termination_type_code],
+        mthly_support_from_vet_amt: dependent[:living_expenses_paid_amount]
+      }
+    end
```

Comment:
It's kind of unorthodox to me that a rails model wouldn't have any attributes.

Here's an example that uses a model initializer to de-serialize a response object into model attributes - [link](https://github.com/department-of-veterans-affairs/vets-api/pull/4574/files#diff-a8bf1190d48d7f74fe39404ce0ed02d7R30-R47) that might be applicable here instead.

Reply:
I hear you, that's great thank you for sharing that resource.